socorro/signature/signature_utilities.py

```diff
@@ -2,31 +2,62 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from itertools import islice
+import logging
 import re
 
 import ujson
-from itertools import islice
 
-from configman import Namespace, RequiredConfig
-from configman.converters import class_converter
+from socorro import siglists
+from socorro.lib.treelib import tree_get
+
 
-from socorro.lib.transform_rules import Rule
+SIGNATURE_MAX_LENGTH = 255
+MAXIMUM_FRAMES_TO_CONSIDER = 40
+SIGNATURE_ESCAPE_SINGLE_QUOTE = True
```

Comment:
These three things used to be configurable, but we don't use non-default values, so I codified them as constants.